docs/source/generated/paasta_tools.spark_tools.rst

```diff
@@ -0,0 +1,7 @@
+paasta_tools.spark_tools module
+===============================
+
+.. automodule:: paasta_tools.spark_tools
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
what's the deal with this?

Reply:
I think that was auto-generated from `make docs`.

Reply:
I think it was generated when he ran `make docs`.